code/game/striketeams/Judge

```diff
@@ -0,0 +1,162 @@
+//Judge
+
+var/list/response_team_members = list()
+
+/datum/striketeam/Judge
+	striketeam_name = Judge
+	faction_name = "Nanotrasen"
+	mission = "preside over this case and come up with an equitable decision."
+	team_size = 1
+	min_size_for_leader = 0//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	spawns_name = "Judge"
+	can_customize = TRUE
+	logo = "Judge-logo"
+	outfit_datum = /datum/outfit/striketeam/judge
+
+/datum/striketeam/Judge/failure()
+	command_alert(/datum/command_alert/judge_fail)
+
+/datum/striketeam/Judge/extras()
+	command_alert(/datum/command_alert/ert_success)
+
+/datum/striketeam/Judge/greet_commando(var/mob/living/carbon/human/H)
+	if(H.key == leader_key)
```

Comment:
given that there's a team size of 1 and min size for leader 0, this should always be true?